### DISCUSSION TITLE
source file and member not showed

### DISCUSSION THREAD

**Discussion by basim1971** [codefori/vscode-ibmi#1197] [IBM i]
Dear All,
Sorry to bother you . After installed the new release i noticed that after i login to my system the list of source file appear as happen in the past but if i try to collapse nothing is shown. I need to click the refresh butto and at this point all the member are shown under the filter 
Is this a normal behavior now ?

Thanks 
Basilio Merlino

**Comment by worksofliam** [comment]
That is odd. I will play around with it and see what I can find.

**Comment by basim1971** [comment]
Do  you have find something ?
